### PR TITLE
Search and filters

### PR DESCRIPTION
**Pull request by anna-yeshtukova**

### CODE REVIEW

File: src/components/filters/_filters.scss [hunk 1]
```diff
@@ -0,0 +1,219 @@
+.nsw-filters {
```
Comment (cdbenj) [inline src/components/filters/_filters.scss:1]: focus ring on the filter button isn't styled
Screen Shot 2021-07-06 at 9 54 07 am

File: src/components/forms/_forms.scss [hunk 2]
```diff
@@ -1,338 +1,387 @@
...
-
-* + .nsw-form {
   @include nsw-spacing(margin-top, xxl);
-}
 
-.nsw-form-label {
-  @include font-stack(heading);
-  @include font-size('sm');
-  display: block;
+  &:first-child {
```
Comment (cdbenj) [inline src/components/forms/_forms.scss:4]: Seems like `@include nsw-spacing(margin-top, xxl);` and `&:first-child { margin-top: 0 }` are offsetting each other in most scenarios. Are there any scenarios where these rules are needed? We can probably omit this if there are none.

File: src/components/forms/_form-default.hbs [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+</div>
+{{/_docs-example}}
+
+<h4>Predictive list</h4>
+{{#>_docs-example}}
+<div class="nsw-form">
+  <div class="nsw-form__group">
+    <label class="nsw-form__label" for="form-predictive-text-1">Suburb</label>
+    <div class="nsw-form__predictive">
+      <input class="nsw-form__input" type="text" id="form-predictive-text-1" name="form-predictive-text-1" value="Horn">
```
Comment (cdbenj) [inline src/components/forms/_form-default.hbs:47]: add `aria-autocomplete="list"` and would be good to turn off the browser's autocomple so it doesn't clash this `autocomplete="off"`

File: src/components/filters/_filters.scss (hunk 1 above)
Comment (anna-yeshtukova) [reply src/components/filters/_filters.scss:1]: Fixed.

File: src/components/forms/_form-default.hbs (hunk 3 above)
Comment (anna-yeshtukova) [reply src/components/forms/_form-default.hbs]: Agreed and done.

File: src/components/forms/_forms-prev.scss [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+  &::after {
+    @include border-radius('round');
+  }
+}
+
+.nsw-form-helper {
+  @include font-stack;
+  @include font-size('xs');
+  display: block;
+  margin-bottom: rem(4);
```
Comment (cdbenj) [inline src/components/forms/_forms-prev.scss:235]: `rem(4px)`

**Review by cdbenj — APPROVED** [review]
LGTM

**Pull request merged**